.github/workflows/push.yml

```
@@ -1,192 +1,37 @@
-              "name": "v${{ steps.version-change.outputs.version }}",
-              "draft": false,
-              "prerelease": false,
-              "generate_release_notes": true
-              }'
+    uses: gatici/.github/.github/workflows/tag-github.yml@add-workflows
 
   release-image:
-    runs-on: ubuntu-latest
     needs: tag-github
-    if: needs.tag-github.outputs.changed == 'true'
-    env:
-      REGISTRY: docker.io
-      DOCKER_REGISTRY: docker.io/
-      DOCKER_REPOSITORY: omecproject/
-    steps:
-      - uses: actions/checkout@v4
-
-      - uses: actions/setup-go@v5
-        with:
-          go-version-file: 'go.mod'
-
-      - uses: docker/login-action@v3.4.0
-        with:
-          registry: ${{ env.REGISTRY }}
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_PASSWORD }}
-
-      - name: Build and push release Docker image
-        env:
-          DOCKER_TAG: rel-${{ needs.tag-github.outputs.version }}
-        run: |
-          make docker-build
-          make docker-push
+    uses: gatici/.github/.github/workflows/release-image.yml@add-workflows
+    secrets: inherit
 
   update-version:
-    runs-on: ubuntu-latest
     needs: tag-github
-    if: needs.tag-github.outputs.changed == 'true'
-    steps:
-      - uses: actions/checkout@v4
-
-      - name: Increment version
-        run: |
-          version=${{ needs.tag-github.outputs.version }}
-          IFS='.' read -r major minor patch <<< "$version"
-          patch_update=$((patch+1))
-          NEW_VERSION="$major.$minor.$patch_update-dev"
-          echo $NEW_VERSION > VERSION
-          echo "Updated version: $NEW_VERSION"
-
-      - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v7
-        with:
-          token: ${{ secrets.GH_OMEC_PAT }}
-          commit-message: Update version
-          committer: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
-          author: ${{ github.actor }} <${{ github.actor_id }}+${{ github.actor }}@users.noreply.github.com>
-          signoff: true
-          branch: version-update
-          delete-branch: true
-          title: Update version
-          body: |
-            Update VERSION file
-          add-paths: |
-            VERSION
+    uses: gatici/.github/.github/workflows/update-version.yml@add-workflows
+    secrets: inherit
 
   branch-release:
-    runs-on: ubuntu-latest
     needs: tag-github
-    if: (needs.tag-github.outputs.changed == 'true') && (needs.tag-github.outputs.release_branch == 'true')
-    env:
-      GITHUB_TOKEN: ${{ secrets.GH_OMEC_PAT }}
-    steps:
-      - uses: actions/checkout@v4
-
-      - uses: peterjgrainger/action-create-branch@v3.0.0
-        with:
-          branch: "rel-${{ needs.tag-github.outputs.version_branch }}"
-          sha: '${{ github.event.pull_request.head.sha }}'
+    uses: gatici/.github/.github/workflows/update-version.yml@add-workflows
+    secrets: inherit
```

Please add end of line
```suggestion
    secrets: inherit

```